file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs

```diff
@@ -446,7 +457,9 @@ public class NewBatchFilesViewModel
         public string MimeType => newBatchFile.MimeType;
 
         public bool CorrectNumberOfFilesFound => ExpectedFileCount == Files.Count();
-        
+        public IEnumerable<KeyValuePair<string, string>> Attributes =>
+        //newBatchFile.Attributes.Select(kv => new KeyValuePair<string, string>(kv.Key, ExpandMacros(kv.Value)));
+        newBatchFile.Attributes.Select(kv => new KeyValuePair<string, string>(kv.Key, kv.Value));
```

Comment:
If you decide to reinstate the `ExpandMacros()` call then I think it would be best to evaluate/populate the `Attributes` property up-front in the class constructor to avoid repeated calls to `ExpandMacros()` during UI binding

Reply:
Macros related changes will be taken care in the PBI 11863, so removing this commented line.